src/alvarium/annotators/handler/contracts.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+from enum import Enum
+class DerivedComponent(Enum):
+    Method = "@method"
+    TargetURI = "@target-uri"
+    RequestTarget = "@request-target"
+    Authority = "@authority"
+    Scheme = "@scheme"
+    Path = "@path"
+    Query = "@query"
+    QueryParams = "@query-params"
+
+    def __str__(self) -> str:
+        return f'{self.value}'
+
+class Constants(Enum):
+    HttpRequestKey = "HttpRequestKey"
+    ContentType = "Content-Type"
+    ContentLength = "Content-Length"
+
+    def __str__(self) -> str:
+        return f'{self.value}'
```

Comment:
I don't think these are `Enum`s, if there is no other way to define constants in Python then maybe the annotation name `HTTPPKI` in `contracts/annotations` can be used to reference the value within the incoming Context that corresponds to the request we need to validate instead of `HttpRequestKey`, just like we do in `tls annotator`. I know that's not how its implemented in Go but Go is more flexible in defining types.

As for `ContentType` and `ContentLength`, I think we can address their values directly.